src/components/screens/Authenticator/modules/RegularPassword/RegularPassword.view.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <form @submit.prevent="onSubmit">
+    <v-title>
+      {{ $t('components.regularPasswordForm.title') }}
+    </v-title>
+    <form-item>
+      <v-input
+        v-model="password"
+        v-validate="'required|min:8'"
+        data-vv-as="password"
+        data-vv-name="password"
+        :error="errors.first('password')"
+        name="password"
+        type="password"
+        required
+        :placeholder="$t('components.regularPasswordForm.placeholder')"
+        data-test="password-input"
+      />
+    </form-item>
+    <form-item class="v-mb-24">
+      <v-button
+        :disabled="!isFormValid"
+        type="submit"
+        data-test="submit-button"
+      >
+        {{ $i18n.t('global.login') }}
+      </v-button>
+    </form-item>
+    <form-row class="v-fs-14 v-text-center">
+      <v-link
+        href="#"
```

Comment:
This not relates to current PR context, but you should not define any href if you are using link as button. Just add `role="button"` for better UX.